server/app/api.py

```diff
@@ -672,7 +672,7 @@ def get_backups(self, obj, user, data):
         return obj.get_backups(data['assignment'], data['quantity'])
 
     def get_submissions(self, obj, user, data):
-        return [subm.submission for subm in obj.get_submissions(data['assignment'], data['quantity'])]
+        return obj.get_submissions(data['assignment'], data['quantity'])
```

Comment:
Mhm. This could actually break master. I made the correction above to fix student and admin bugs with submission views. Each submission has a `backup` property, so if you really need to, you should use `[subm.backup for sum in get_submissions(...)`.